src/magic_folder/test/cli/test_magic_folder.py

```diff
@@ -220,21 +217,6 @@ def test_list_some_json(self):
         )
 
 
-def addOutcomeDetails(testcase, outcome):
```

Comment:
This is an unused bit of code that does a similar thing via testtools, instead of eliot.